common/changes/@uifabric/example-app-base/tooltipHost-onTooltipToggle_2017-08-11-16-12.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "",
+      "packageName": "@uifabric/example-app-base",
+      "type": "none"
```

Comment:
Thank you, I never remember from one time to another, I guess I got my command from the create PR template and thought `none`s were expected. Will clean up.